testing/scenario_app/ios/Scenarios/Scenarios/AppDelegate.m

```diff
@@ -1,12 +1,46 @@
 #include "AppDelegate.h"
+#import "TextPlatformView.h"
+
+@interface NoStatusBarFlutterViewController : FlutterViewController
+
+@end
+
+@implementation NoStatusBarFlutterViewController
+- (BOOL)prefersStatusBarHidden {
+  return YES;
+}
+@end
 
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication*)application
     didFinishLaunchingWithOptions:(NSDictionary*)launchOptions {
   self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
-  self.window.rootViewController = [[UIViewController alloc] init];
+
+  if ([[[NSProcessInfo processInfo] arguments] containsObject:@"--platform-view"]) {
```

Comment:
Can you add a comment here for when this argument would be expected?

Reply:
Done